_data/oscar_version.yml

```diff
@@ -0,0 +1,2 @@
+latest_release: "1.3.0"
+latest_release_date: "28 February 2025"
```

Comment:
I don't want to bikeshed, but if you only ever need the year, why not restrict to that? Or even if you sometimes need the year and sometimes the full date, perhaps just store both in this file (the plan is to update this file by a script anyway, right? so redundancies won't matter)

Also if you rename the file to e.g. `release` (or `latest_release`?) and the keys in here to `version`, `date`, `year`, then you can replace `site.data.oscar_version.latest_release` by `site.data.release.version`, and also use ``site.data.release.date` and ``site.data.release.year` which IMHO is nicer (less verbose and still more on the point).

But again, this is bikeshedding, I am also fine with the current format 🤷 😄

Reply:
Great suggestion. Executed.